notification/__init__.py

```diff
@@ -82,3 +82,9 @@ def get_absolute_url(self) -> Optional[str]:
         Returns the absolute url of the entity.
         """
         return None
+
+    def display_name(self) -> str:
+        """
+        Returns the display name of the entity.
+        """
+        return self.get_absolute_url()
```

Comment:
Wasn't it supposed to be self.__str__() by default?

Reply:
You are absolutely correct. This is been addressed in the latest commit